en/services/multi_gcs.md

```diff
@@ -0,0 +1,128 @@
+# Multi GCS protocol
+## Introduction
+A controlled vehicle should only accept MAVLink commands (or command-like messages) that are sent by the controlling GCS, or from other components with the same system id. Commands from other system should be rejected with MAV_RESULT_PERMISSION_DENIED (except for this command, which may be acknowledged with MAV_RESULT_ACCEPTED if control is granted).
+
+The command is sent by a GCS to the autopilot component to request or release control of a system, specifying whether subsequent takeover requests from another GCS are automatically granted, or require permission.
+The autopilot should grant control to the GCS if the system does not require takeover permission (or is uncontrolled) and ACK the request with MAV_RESULT_ACCEPTED.
+
+The autopilot should stream CONTROL_STATUS indicating its controlling system: all other components with the same system id should monitor this message and set their own owner to match that of the autopilot.
+If the autopilot cannot grant control (because takeover requires permission), the request should be rejected with MAV_RESULT_PERMISSION_DENIED.
+
+The autopilot should then send this same command to the current owning GCS in order to notify of the request.
+The owning GCS would ACK with MAV_RESULT_ACCEPTED, and might choose to release ownership of the vehicle, or re-request ownership with the takeover bit set to allow permission. 
+
+Note that the pilots of both GCS should co-ordinate safe handover offline.
+It is also possible for a GCS to request control of a specific component rather than the whole system (though this is not recommended).
+In this case the command should be sent to the specific component, which should also emit CONTROL_STATUS to indicate its owner (this component would ignore the CONTROL_STATUS set by the autopilot).
+The flow is otherwise the same.
```

Comment:
We also need to add that:
- System can be uncontrolled
- If the autopilot ('controlling component') loses its connection to the controlling GCS we need to make it clear **how** the ownership lapses. 
  - IMO we don't want a GCS that has been turned off to be the controller, but nor do we want a vehicle that flies out of range and then starts to return to have no owner (though this is not critical as a vehicle will accept commands from any GCS if it has not owner)
  - So I "think" maybe a model like: after short connection break, set takeover allowed. After a bit longer, make uncontrolled.